addons/im_livechat/report/im_livechat_report_channel_views.xml

```diff
@@ -64,6 +64,11 @@
             <field name="res_model">im_livechat.report.channel</field>
             <field name="view_mode">graph,pivot</field>
             <field name="context">{"graph_measure": "time_to_answer", "search_default_last_week":1}</field>
+            <field name="help" type="html">
+                <p class="o_view_nocontent_smiling_face">
+                    No data yet !
```

Comment:
Just a quick note: there is never a space before the punctuation at the end of a sentence in English.
Typically error because we need them in French before ? ! and :

Reply:
Not me but mba and he is not french speaking :-)

Reply:
Apparently there is a story about this.

> In most Indian schools, English grammar is taught from a book that was originally authored by two Englishmen: PC Wren and H Martin. The earlier editions of the book (including the one that I own) has a space before the following punctuation marks are used: question mark, exclamation mark, dash (em dash), colon and semi-colon. Also, one can observe two spaces after a full-stop (or a "period" as the Americans call it). My guess is that such practice is a relic of the typewriter era. The new editions of "Wren & Martin Grammar Book" (as it is known in India) appear not to have such spaces. – user36421

https://english.stackexchange.com/questions/4645/is-it-ever-correct-to-have-a-space-before-a-question-or-exclamation-mark#comment206109_4645

Reply:
@seb-odoo , @Polymorphe57 , Yes I am not french speaker but the Excel sheet was given by the NGR with all the help content messages. Thanks